shoop/admin/modules/orders/views/shipment.py

```diff
@@ -65,6 +65,7 @@ def get_form(self, form_class):
             if info["unshipped"] == 0:
                 attrs["disabled"] = "disabled"
             field = forms.DecimalField(
+                required=bool(info["unshipped"] != 0),
```

Comment:
`info["unshipped"]` is used 4 times (atleast) in preceeding lines. I would like to see it being extracted once from `info`. for example `is_unshipped = info["unshipped"]`